docker-compose-state-migration.yml

```diff
@@ -356,7 +356,7 @@ services:
 
 networks:
   network-node-bridge:
-    name: network-node-bridge
+    name: hedera-network-node-bridge
```

Comment:
maybe for consistency's sake all networks should have a -bridge suffix, or none of them

Reply:
@isavov 
I chose the latter option since there is only one network with the “bridge” suffix and a few without it.

I have only updated the name used by Docker, not the Docker Compose level alias, as changing the alias seemed to be outside the current scope.